aiida/manage/tests/pytest_fixtures.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""
+Collection of pytest fixtures using the TestManager for easy testing of AiiDA plugins.
+
+ * aiida_profile
+ * clear_database
+ * aiida_localhost
+ * aiida_local_code_factory
+
+"""
+from __future__ import division
+from __future__ import absolute_import
+from __future__ import print_function
+
+import tempfile
+import shutil
+import pytest
+from aiida.manage.tests import test_manager, get_test_backend
+
+
+@pytest.fixture(scope='session', autouse=True)
+def aiida_profile():
+    """Set up AiiDA test profile for the duration of the tests.
+
+    Note: scope='session' limits this fixture to run once per session. Thanks to ``autouse=True``, you don't actually
+     need to depend on it explicitly - it will activate as soon as you import it in your ``conftest.py``.
+    """
+    # create new TestManager instance
+    with test_manager(backend=get_test_backend()) as test_mgr:
```

Comment:
We either need to configure the logger here, or find the correct place of doing it in the manager.
In any case, in this setup I don't think the logging will be configured for the first test.
```
with test_manager(backend=get_test_backend()) as test_mgr:
    from aiida.common.log import configure_logging
    configure_logging(with_orm=True)
    yield test_mgr
```

Reply:
Thanks for the hint - I'll do this in the `test_manager` contextmanager directly.